admin/lib/solidus_admin/engine.rb

```diff
@@ -16,6 +17,12 @@ class Engine < ::Rails::Engine
 
     initializer "solidus_admin.view_component" do |app|
       app.config.view_component.preview_paths << SolidusAdmin::Engine.root.join("spec/components/previews").to_s
+
+      app.config.to_prepare do
+        preview_controller_class = app.config.view_component.preview_controller.constantize
+
+        preview_controller_class.include SolidusAdmin::Preview::ControllerHelper
```

Comment:
I'm failing to understand why we need to include that module in the view component's controller. Can you help me understand?

Reply:
> So, for instance, in order to have current_component available to both inline-renders (from within the preview class) and inside preview templates (that run in the context of the controller) we need to add the method to two different places.

I see. Thanks for the clarification. Would it be good to add a comment about that, or otherwise in the commit message?